tests/unit_tests/unit_monetization_api.test.ts

```diff
@@ -621,7 +621,7 @@
                             tierType: 'sortByCpm',
                             instances: [
                                 {
-                                    providerName: 'Vungle',
+                                    providerName: 'liftoffMonetize',
```

Comment:
The providerName has been changed from 'Vungle' to 'liftoffMonetize'. Please make sure this is the intended change and update any other relevant code accordingly.